.github/workflows/gitleaks.yaml

```diff
@@ -69,5 +69,14 @@ jobs:
         with:
           configuration-data: |
             threshold: 0.95
+            ignore: |
+              .git
+              man/figures/*.svg
+              man/figures/*.png
+              DESCRIPTION
```

Comment:
I expect, that this file should be checked

Reply:
I have removed `DESCRIPTION` from ignore statement.